lib/alephant/sequencer.rb

```diff
@@ -1,14 +1,16 @@
 require "alephant/sequencer/version"
 require "alephant/sequencer/sequencer"
 require "alephant/sequencer/sequence_table"
+require "alephant/sequencer/sequence_cache"
 
 module Alephant
   module Sequencer
     @@sequence_tables = {}
 
-    def self.create(table_name, ident, jsonpath = nil, keep_all = true)
+    def self.create(table_name, ident, jsonpath = nil, keep_all = true, config = {})
```

Comment:
This method signature is starting to get a bit long winded. Be good to see if we can refactor this within this PR (e.g. consolidate the 2nd, 3rd, 4th and the new 5th argument)

So it looks something like:

```rb
self.create(table_name, opts)
```

Where `opts` would be a hash like:

```rb
{
  :ident    => foo,
  :jsonpath => bar,
  :keep_all => baz,
  :config   => {}
}
```